static/index.html

```diff
@@ -176,24 +167,24 @@
                     <label>CI<br /><input type="url" id="resources_ci" size="40" /></label> Continuous Integration (e.g. Jenkins) Server where the module is being built
                 </li>
                 <li>
-                    <label>SpaceDock<br /><input type="url" id="resources_spacedock" size="40" /></label> The mod on SpaceDock
+                    <label>SpaceDock<br /><input type="url" id="resources_spacedock" size="40" /></label> The mod's SpaceDock page
                 </li>
                 <li>
-                    <label>Manual<br /><input type="url" id="resources_manual" size="40" /></label> The mod's bugtracker if it exists
+                    <label>Manual<br /><input type="url" id="resources_manual" size="40" /></label> The mod's user manual if it exists
                 </li>
                 <li>
-                    <label>License<br /><input type="url" id="resources_license" size="40" /></label> The mod's license
+                    <label>License<br /><input type="url" id="resources_license" size="40" /></label> A URL to the mod's license
                 </li>
                 <li>
-                    <label>Screenshot<br /><input type="url" id="resources_x_screenshot" size="40" /></label>
+                    <label>Screenshot<br /><input type="url" id="resources_x_screenshot" size="40" /></label> A direct URL to a screenshot of the mod
                 </li>
             </ol>
         </div>
-        <h3>Special Install</h3>
+        <h3>Install instructions</h3>
         <div id="install_fields">
-            <ol>
+            <ol> If this mod requires other mods to be installed alongside it, use Relationships.<b>Do not use install instructions to install multiple mods.</b>
```

Comment:
This line in particular probably needs an edit since I imagine this text should be inside some kind of element. *shrug*